cwl/src/main/scala/wdl4s/cwl/AddEmbeddedCwl.scala

```diff
@@ -0,0 +1,50 @@
+package wdl4s.cwl
+
+import shapeless.{Path => _, _}
+import ammonite.ops._
+import cats.syntax.traverse._
+import cats.syntax.either._
+import cats.instances.list._
+import cats.effect.IO
+import CwlDecoder.{Parse, ParseValidated}
+import cats.{Applicative, Monad}
+import cats.data.EitherT._
+import cats.data.EitherT
+import lenthall.validation.ErrorOr._
+
+object AddEmbeddedCwl extends Poly1 {
+  val workflowStepLens = lens[Workflow].steps
+  val workflowStepRunLens = lens[WorkflowStep].run
+
+  implicit def workflow =
```

Comment:
The actual return type is pretty gross and just as hard to understand :
![image](https://user-images.githubusercontent.com/165320/29632148-06173590-8810-11e7-86fe-14200a8c2d26.png)